notebooks/00_extract_coordinate_images.ipynb

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Please make sure your notebooks are sequentially executed starting with cell [1]. (It looks like this one is out of order)